app/src/main/java/com/endava/myendava/adapters/EventAdapter.java

```diff
@@ -0,0 +1,96 @@
+package com.endava.myendava.adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.endava.myendava.R;
+import com.endava.myendava.models.Event;
+
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class EventAdapter extends RecyclerView.Adapter<EventAdapter.EventViewHolder> {
+
+    public static final String TRAINING_EVENT = "Training";
```

Comment:
duplicate code ... it exist also in the other adapter. please extract in from the adapters in a common place